rules/windows/process_creation/proc_creation_win_reg_write_protect_for_storage_disabled.yml

```diff
@@ -1,10 +1,14 @@
 title: Write Protect For Storage Disabled
 id: 75f7a0e2-7154-4c4d-9eae-5cdb4e0a5c13
 status: test
-description: Looks for changes to registry to disable any write-protect property for storage devices. This could be a precursor to a ransomware attack and has been an observed technique used by cypherpunk group.
+description: |
+    Detects application trying to modify the registry in order to disable any write-protect property for storage devices.
+    This could be a precursor to a ransomware attack and has been an observed technique used by cypherpunk group.
```

Comment:
```suggestion
    Detects applications trying to modify the registry in order to disable any write-protect property for storage devices.
    This could be a precursor to a ransomware attack and has been an observed technique used by cypherpunk group.
```